src/main/java/com/minecolonies/coremod/commands/generalcommands/RandomTeleportCommand.java

```diff
@@ -154,7 +154,7 @@ private static void teleportPlayer(final ICommandSender sender, final EntityPlay
 
             final Colony colony = ColonyManager.getClosestColony(sender.getEntityWorld(), tpPos);
             /* Check for a close by colony*/
-            if (BlockPosUtil.getDistance2D(colony.getCenter(), tpPos) < Configurations.gameplay.workingRangeTownHall * 2 + Configurations.gameplay.townHallPadding)
+            if (colony != null && BlockPosUtil.getDistance2D(colony.getCenter(), tpPos) < Configurations.gameplay.workingRangeTownHall * 2 + Configurations.gameplay.townHallPadding)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 181 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00103)

Reply:
I still don't understand why you would square the players/admin Config setting for workingRangeTownHall. I understand squaring coordinates and subtracting the squares to then get the squareroot to get a distance between the two. However, the Config value is already a distance, so why are you squaring it?